lib/spoom/counters.rb

```diff
@@ -0,0 +1,30 @@
+# typed: strict
+# frozen_string_literal: true
+
+module Spoom
+  class Counters
+    #: Hash[String, Integer]
+    attr_reader :counters
```

Comment:
I wonder if we should just inherit from `Hash`, so that you can invoke `sort_by` directly on the object.